src/libraries/Common/tests/System/DirectoryServices/LDAP.Configuration.xml

```diff
@@ -83,5 +83,14 @@ Note:
         <Password>%TESTPASSWORD%</Password>
         <AuthenticationTypes>ServerBind,None</AuthenticationTypes>
     </Connection>
+    <Connection Name="SLAPD OPENLDAP SERVER TLS">
+        <ServerName>localhost</ServerName>
+        <SearchDN>DC=example,DC=org</SearchDN>
+        <Port>1636</Port>
+        <User>cn=admin,dc=example,dc=org</User>
+        <Password>admin</Password>
```

Comment:
We have a scanning tool that looks for things that look like secrets. Apparently it's not finding `password` (in this obviously non standard format file). You might use `password` here just in case `admin` trips it. Or more to the point, to make it clearer that it's not a real password. (I hope!)

Reply:
Updated to use `password` (also reflected in test docs), though I don't know how to check what you're seeing.